static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -676,14 +676,31 @@ CStudioAuthoring.Module.requireModule(
           return inputEl;
         },
 
+        /**
+         * Check input with space only
+         */
+        _isInputSpace: function (obj) {
+          if (!obj.validateTextArea) {
+            obj.validateTextArea = document.createElement('textarea');
+          }
+          obj.validateTextArea.innerHTML = obj.value;
+
+          if (!obj.validateDiv) {
+            obj.validateDiv = document.createElement('div');
+          }
+          obj.validateDiv.innerHTML = obj.validateTextArea.value;
+
+          return obj.validateDiv.innerText.trim() == '';
+        },
```

Comment:
- What's exactly is this method's purpose and the need for it? Why not directly `obj.value.trim()`?
- What's the purpose of storing the temp validateTextArea/validateDiv?